emcc.py

```diff
@@ -1599,7 +1596,10 @@ def phase_linker_setup(options, state, newargs, user_settings):
     # configure tests want a more shell-like style, where we emit return codes on exit()
     settings.EXIT_RUNTIME = 1
     # use node.js raw filesystem access, to behave just like a native executable
-    settings.NODERAWFS = 1
+    if settings.MEMORY64:
+      settings.ENVIRONMENT = 'shell'
```

Comment:
I guess this is necessary for now, as there isn't memory64 in node yet? Worth a TODO here as we'll want to remove this eventually.

Reply:
Actually I ended up not need this part.  Reverted.